dedibox-network/rpn/reference-content/private-networks-overview.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Private Networks on Scaleway Dedibox and Scaleway Elements
-description: This page explains the different private network types and offers available at Scaleway on Dedibox and Elements, and compares their different capabilities and features.
+description: This page explains the different Private Network types and offers available at Scaleway on Dedibox and Elements, and compares their different capabilities and features.
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
description: This page explains the different private network types and offers available at Scaleway on Dedibox and Elements, and compares their different capabilities and features.
```